docs/docs-content/enterprise-version/install-palette/install-on-kubernetes/install.md

```diff
@@ -133,6 +133,14 @@ your environment. Reach out to our support team if you need assistance.
     | `ingress.enabled`                         | Whether to install the Nginx ingress controller. Set this to `false` if you already have an Nginx controller deployed in the cluster.                          | boolean  |
     | `reach-system`                            | Set `reach-system.enabled` to `true` and configure the `reach-system.proxySettings` parameters to configure Palette to use a network proxy in your environment | object   |
 
+    :::info
```

Comment:
@lennessyy we will need this also in the airgap install version. Your call, but I think we can reduce the admonitions and just make this an entry in the table.

Reply:
I thought about using the table, but i decided to isolate it with the admonition because i feel like this is a pretty niche use case and most people don't need it. The paragraph before the table says the parameters in it are required, so i thought it'd leave it with the essentials and pull this one out.